### PR TITLE
TECHPUBS-4214: Corrected references to Slingshot documentation (1.5)

### PR DESCRIPTION
**Pull request by nrockershousen**
# Description

Fixing several section and document titles for Slingshot.
Moving Slingshot switch firmware update to correct IUF stage.

https://jira-pro.it.hpe.com:8443/browse/TECHPUBS-4214

Relates to:
- https://github.com/Cray-HPE/docs-csm/pull/5035
- https://github.com/Cray-HPE/docs-csm/pull/5038

# Checklist

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [ ] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Review by kgarima123 — APPROVED** [review]

**Review by rustydb — APPROVED** [review]

**Comment by rustydb** [comment]
Waiting on merging this one https://github.com/Cray-HPE/docs-csm/pull/5045

File: operations/iuf/workflows/configuration_of_SFM.md [hunk 1]
```diff
@@ -7,10 +7,12 @@ This section configures the Slingshot Fabric Manager.
 
 ## 1. Manual configuration of the Slingshot Fabric Manager
 
-Instructions to configure the Slingshot Fabric Manager are provided in the "Slingshot Installation in a Kubernetes Orchestrated Container Environment" section of the _Slingshot Operations Guide for Customers_. Follow all
-subsections beginning with "Configure Fabric Manager" and ending with "Verify Fabric Manager version".
+Instructions to configure the HPE Slingshot Fabric Manager are provided
+in the “Install Slingshot software in a CSM environment” section of the _HPE Slingshot Operations Guide_.
```
Comment (mharding-hpe) [inline operations/iuf/workflows/configuration_of_SFM.md]: This PR already merged into 1.6, so I'm fixing this with https://github.com/Cray-HPE/docs-csm/pull/5064. But we can fix it here for 1.5, before it merges.

```suggestion
in the "Install Slingshot software in a CSM environment" section of the _HPE Slingshot Operations Guide_.
```

File: operations/iuf/workflows/configuration_of_SFM.md [hunk 2]
```diff
@@ -7,10 +7,12 @@ This section configures the Slingshot Fabric Manager.
...
+Follow all subsections up to the section titled "Install or upgrade Slingshot Host Software".
 
-For systems with Slingshot NICs, also follow the instructions in the "Fabric Configuration for Slingshot NICs" section.
+For systems with Slingshot NICs, also follow the instructions in
+the “Installing 200Gbps NIC host software” section of the _HPE Slingshot Operations Guide_.
```
Comment (mharding-hpe) [inline operations/iuf/workflows/configuration_of_SFM.md]: This PR already merged into 1.6, so I'm fixing this with https://github.com/Cray-HPE/docs-csm/pull/5064. But we can fix it here for 1.5, before it merges.

```suggestion
the "Installing 200Gbps NIC host software" section of the _HPE Slingshot Operations Guide_.
```

**Review by mharding-hpe — CHANGES_REQUESTED** [review]
The two linting comments I made should be addressed before merging

**Comment by rustydb** [comment]
> The two linting comments I made should be addressed before merging

@mharding-hpe, there were two backports that merged last week; this remained open due to two merge conflicts. Do we need follow-up PRs to those other base branches?

**Comment by rustydb** [comment]
> > The two linting comments I made should be addressed before merging
> 
> @mharding-hpe, there were two backports that merged last week; this remained open due to two merge conflicts. Do we need follow-up PRs to those other base branches?

nevermind, I see your comments have remarks mentioning you fixed 1.6 in https://github.com/Cray-HPE/docs-csm/pull/5064

**Pull request merged**